README.md

```diff
@@ -9,39 +7,108 @@
+Terasology blog posts are authored in [GitHub markdown](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) and typically cover relatively neutral news related to releases, recent contributions or contributor programs.
 
----
+> **⚠️ NOTE:** If you write an opinionated post about your personal experiences and views on Terasology or related topics, please add a disclaimer at the top of your blog post indicating this.
 
-<h2> Getting Started </h2>
+All blog posts are stored in the `/blog` directory with a dedicated subdirectory for each of them.
+Every blog post directory name consists of the publish date and blog post title, e.g. `2017-07-14-gsoc-2017-an-overview`
+Blog post directory names should not contain non-alphanumeric characters other than dashes (`-`)
 
-The module website development environment can be installed on macOS, Windows, and Linux (Debian or Ubuntu recommended). You’ll need at least 1GB of available RAM.
+Blog post directories should always contain a markdown file named `index.md` which will hold the blog post content and metadata.
+Additionally, you can provide a cover image in jpg format named `cover.jpg`.
+Optionally, any images you want to include into the blog post should be in jpg format and included in the blog post directory.
 
+### Blog Post Template
+
+To allow your blog post to show up properly on the website and search results, you'll need to use the following template in your `index.md`:
+
+```
+---
+posttype: blog
+title: "My first Terasology blog!"
+cover: "./cover.jpg"
+description: "This blog is about my latest experience with Terasology"
+author: "It's me, Mario!"
+date: "2023-04-08"
+tags: ["Announcement"]
 ---
 
-## Prerequisites
+Hello world, this is my newest blog post for Terasology!
+I'm so excited to share this with you!
 
-This website is built with [Gatsby](https://www.gatsbyjs.org/).
-The following tools must be installed to build and serve the website locally:
+See below the two wonderful screenshots I took.
+  <div class="row">
+    <div class="col-md-6"> <img src="./image-1.jpg" width="40%" /> </div>
+    <div class="col-md-6"> <img src="./image-2.jpg" width="40%" /> </div>
+  </div>
+
+```
+
+The upper part is metadata that will processed for nicely displaying your blog post on the website.
+THe lower part is your blog post content. You'll also see an example of how to include multiple images side-by-side there.
+
+> **⚠️ NOTE:** All the metadata fields are required.
+
+- `posttype` should always be `blog` (without quotation marks)
+- `title` is your news in a short phrase
+   (if you're writing a TeraSaturday post, please prefix it with `TeraSaturday #<id>: ` where "\<id\>" is the last TeraSaturday post ID + 1)
```

Comment:
Worth noting: The title will define the path under which the blog is being published. For the example above this would be: `blog/my-first-terasology-blog/`, I think.

Observation: we can easily have a duplication in the title, and it is not obvious that there is a uniqueness constraint on it (nor does it make sense, I think). We should think about using the date in combination with the title, for instance.

Reply:
Note added.
Also created an issue for the uniqueness topic: https://github.com/MovingBlocks/movingblocks.github.com/issues/213